src/Microsoft.AspNet.Mvc.Core/DefaultControllerActionArgumentBinder.cs

```diff
@@ -63,6 +63,41 @@ public class DefaultControllerActionArgumentBinder : IControllerActionArgumentBi
             return actionArguments;
         }
 
+        public async Task<ModelBindingResult> BindModelAsync(
```

Comment:
Will add doc comments for the public api surface for this class in a separate pr ( not holding this one for furhter review (as you have already signed off)). Not opening individual issues for adding doc comments, there are tons of places which needs fixing and this could be tracked by a singular issue.